config/config.go

```diff
@@ -0,0 +1,76 @@
+package config
```

Comment:
In your case this config wrapper is unnecessary, you can use directly a viper config object and use `viper.SetDefault()` for setting the default values